app/models/jellyfish_demo/product_type/demo.rb

```diff
@@ -0,0 +1,34 @@
+module JellyfishDemo
+  module ProductType
+    class Demo < ::ProductType
+      def self.load_product_types
+        return unless super
+        transaction do
+          [
+            set('Demo Compute', '6e1c01ab-64e9-43b0-891f-5657170c845d', description: 'Demo Compute Product Type'),
```

Comment:
Each product should be defined individually, this way each can have it's own list of `product_questions`.